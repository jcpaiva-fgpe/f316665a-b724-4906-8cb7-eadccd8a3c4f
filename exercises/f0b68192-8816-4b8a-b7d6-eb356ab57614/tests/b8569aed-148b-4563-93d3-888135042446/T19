30 255
000101101010000010001010011111010000111000100100000111111100001011101011001111111100000110111000010111011101010101011000001101100011111101001001100001001001110011111001000100000100010110000001000100000011001011010101100110110101001011111000001011011000011
111010111010110111000101100101110011011110011001111001111111101100111001000110001100001110100001010010111100110101111000111111000001110000011000111001111000110110001001001011001111000100011001101000000101000111001011110000110110110101111101011110000010001
010000000111100001110111110110010110110111101101010001100111000011001110000010111001111011111010110001010011001101100001110011011011000001110000110000110001100010010010001010001101101101111100101100001100110101011111000010010001001100111001110001111111010
010101111110011100100111000100111001111000001111111000100001010110100000011100001101110110011011101100101011111001011000010111001011011110001011110010111101011111010101100001100101110100001110011001010010100111010001001011000001100100010000101001111010110
011111100101010000101010101110010101001010011011000101111011110101001110110011010010100111000010110001100101001000100111011010011010000010001101111100101001100011010011111000111110100101011000101111100001010001001010111111100101011011100001101100101010000
010111001111111011101111010011110100100011111010110001010010101000010010101000011010111000011111110011011111100101101001000110011011110001111100110010100111100101000000100110100101111111001111011110110101010000110101001111000010001001100100011001101011101
000000100010010010110001100000100100010010111110101001111101011011110111100001011111001001100101010110000110010011110010110000100011001010110111010010011011101101001110001011110101100011100110011100100001110110101110100111110000011001111010000111100111011
011101001100001001101011011010101011110000110010000110011110100111000011000001100111100010110101100111011001010001101101110010100111001101100100010011100101101111101011100110100001111100110001111011001011011011100110101111001010111010111000110100110001010
111100001100000100110110010111100000001111001011001100101111111111000111011100101011001100011001101110110010100000100010001000110010001000010110100100010110101000000111010000110111110101101101101010110100110001100110010100010110001001010011101110011001101
100101101000110000100101100010110000110001010010100110001010101100011011100101011001111010001001001100100111110001110111110010010110100010101111000011011010010001001100111000010010011110110110000110001100101110110001111110110011011000010101100100110001000
101000000001010010000110110000010011011110001000001110100100011010110101111011100011000010011100010000111101011010010001001001000001110000111010010010000100000010100000001001010010010100100011001000110010111001000011011010011101110111011111011111010111010
110001101000111000011110101001010100010110110010011100110111000010011111100111111011001110010011101010100101111000100010111101111010010000010110110110101101010110111111011000011011001011110001010110011110011110001000110010110001010101100101101001101100110
110011010100110111011001111111100010111111001010101100011000101100111010011110110000001000010110101110111001110110000101100101110101100100011010010000101011110101011001001100001011101001100110001100110001001110001001111101111001010111011100010110001010101
001111000100010111100010111101000011111101110000110011010000110110001110011111100100111111000001001001000111001000011100100110111011111101001011010111101110000111001101000001100011101001101110011101111110000000010100010001000101010001001000001101100000111
001110110010110101111001100101001110101100110010101001110110010011001001010011001100101110100101001101011010001111101111101111011000101010110000100110000100001100100001110111111000000010011001001000011011100000000111110010010101100101000000011000111101100
010110101101101100000010000110111000000110000010000101101111110011111110010011101110100010101000111010010011101100110111111010001000001001011101010010001110110110100100001100101100101000011001001011010000010110001111110001101010011011011010010101011111011
110110011111010001111000111010000011011000110110001111100011111110001100011000001110100101110000110000001101001100110100110100111110100011111001010011100110111101101100000000110011101110011100010100010011000001110011110000011000101100010011010011100011010
000011101110110011110011110011011010101110111111000111000000101101101011000001100111111111111101011100001001011011100010111001100110101001101100010011000100000110100010001111111100001101101010011010010100111001010100110010111010000010000000111101111000010
111111111001000011011101001100111010001101001011110100011101111010010101111010111100001111000101101011100010101001001110010010000101010101111011101010000001001010101111000000110111111110010010000101101101100111100001000101101101010100000011100110010001111
001000011011110001010011110001110110110011101011111100010010001010011011011110000110011000000000001010111000100101101001001001011110001101010000010110010101000101001111111101101110110100000010110010001111001110001011001001100101101110000011011011111101011
000000110010011001100010100101011011100100111011101011001010101000010001010001111110011110100011010111000000110001101000011100001010110010000010001101100010011100010011100101101011111011000110010001111100010100011010100110011011110100110100111011101110110
111010101110100101100010110101111100110001001111000101110111100010011010100011010000100011101101000101111100100100110110000001101110011001011001101110101001111000001000111110000101011111000110111111110000000000000111110101010100011101101111011000111111011
101101011101000001100011110111001110011100110110100101000110010000011100111110100111111100111011010101100011011100010110101000101000110101100011110101110110111111100110001110000010000011010101011101101010110101100101100100101011110111101011111101111111000
101100110000100001111001000100110100111100010000110000100110110000010010011011101101101010011011111110111000010100010110101100011101110000001011111100110001010001100100010111100000010100001000110010011101011111101111011110111110011100011001111010111100101
010110010011011111010011101111111100111011110101001000110011000111110111001011010110100111011000111100101010110101010001100000001010000101000010111111001000010011010010010010101001001110111110111100101000011101001010010101111001101001101011101000101110001
100010100110011101001011000010101110100111110100001010111010000101010011001011100001010111101011111100100000110110001011111110000100110010000001001111000111001011010011010010010101010001101101000101010011010110011000010101011000101110010001110010010110011
101110011110101010011101110111101001000111001100110000111100110000101100101011100100101010001011100000010111101111001101101110001001000001110001000100011000010110000000001000100100000100111101111111011110110000001011100011100001001110111010010100110110010
010010110101100111101110010011000100111101010001001111010011101111100001000011111000100101111010001011111010101000000011000111010111010111001101101011111100101111010011111001101100011100010100001000111010110100101111111111110100110101110100100010101010010
000011001011001111110110110111111100101000111100010111100110011101010100111001111100010011010011110100010100111001001000111010101101001011000000100000001000011001101000111111100110010110001100010101110011001000000111111101101111010011110110100111101011000
101000010001011011000110000000010111010010110011111111011011010111010010001011100101001100000011110111101100001011011101101101010000100110110011110000100110001000000011100111101111001011111001001010101110100001100101011101001010000100110010100111110000111
