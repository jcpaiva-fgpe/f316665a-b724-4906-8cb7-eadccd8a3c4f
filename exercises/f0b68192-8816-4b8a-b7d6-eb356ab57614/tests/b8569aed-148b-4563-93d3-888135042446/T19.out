000110101110010001100011110111110000111110111010000101110110100010011011001011101100101110011001110100111001111001100000101000001010000001011001110010100100000111000001001000100101001100001100011100010000010100000111110111110001011101110000111101111010010
